pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -121,6 +121,7 @@ public void run() {
         server.addServlet("/metrics", new ServletHolder(MetricsServlet.class));
         server.addRestResources("/", VipStatus.class.getPackage().getName(),
                 VipStatus.ATTRIBUTE_STATUS_FILE_PATH, config.getStatusFilePath());
+        server.addServlet("/admin", new ServletHolder(new AdminProxyHandler(config)));
```

Comment:
We should also proxy `/loookup` prefix, for HTTP based topic lookups

Reply:
done